### DISCUSSION TITLE
subscribers get deleted from subscriber db when sync with orchestrator which was added using subscriber_cli.py script not from UI

### DISCUSSION THREAD

**Discussion by logic2code** [magma/magma#7204] [Q&A] [answered]
when I try to add subscribers with /root/magma/lte/gateway/python/scripts/subscriber_cli.py script and not exist in orchestartor UI.
subscribers get deleted after get sync with NMS/Orchestration instead to sync subscriberdb to NMS/Orchestrator
after few seconds
Error! [StatusCode.NOT_FOUND] Subscriber not found: IMSI001010000000001
any workaround for this or other to add then UI?
```
`root@magma:/home/cloud# /root/magma/lte/gateway/python/scripts/subscriber_cli.py get IMSI001010000000001
sid {
  id: "001010000000001"
}
gsm {
}
lte {
  state: ACTIVE
  auth_key: "\000\001\002\003\004\005\006\007\010\t\n\013\014\r\016\017"
}
state {
}
non_3gpp {
  apn_config {
    service_selection: "internet"
    qos_profile {
      class_id: 9
      priority_level: 15
    }
    ambr {
      max_bandwidth_ul: 3000
      max_bandwidth_dl: 4000
    }
  }
}

root@magma:/home/cloud# /root/magma/lte/gateway/python/scripts/subscriber_cli.py get IMSI001010000000001
Error! [StatusCode.NOT_FOUND] Subscriber not found: IMSI001010000000001
root@magma:/home/cloud# /root/magma/lte/gateway/python/scripts/subscriber_cli.py get IMSI001010000000001
Error! [StatusCode.NOT_FOUND] Subscriber not found: IMSI001010000000001`
```

**Comment by ulaskozat** [comment, marked as answer]
Orc8r is the authority on subscriber data. You should not use the subscriber cli if AGW is already controlled and configured by the Orc8r. Such CLI is intended to be used in headless operations (i.e., AGW is not connected to the cloud). The workaround is to disconnect AGW from the Orc8r so that your local configs are not overwritten by the cloud. Note that this is a feature and intended behavior not a bug.

**Reply by pramchan** [reply]
lgtm
